pkg/helmexec/exec.go

```diff
@@ -17,7 +17,7 @@ const (
 	command = "helm"
 )
 
-type execer struct {
+type Execer struct {
```

Comment:
I think you can safely keep this private within the helmexec pkg, as your code is well decoupled by using [`helmexec.Interface`](https://github.com/roboll/helmfile/pull/693/files#diff-637a60a871f8b69622bdfd3d3b2fcad1R49) and there's no direct calls to `Execer{}` outside of `helmexec`.

So, would you mind if I changed the struct name back to `execer` later?

// I'd appreciate it if you could correct me if I'm missing something and this needs to be exposed to other pkgs :)

Reply:
I passed it public to allow create a MockExecer in app_test.go (but I may have missed someting to do better because it my first golang code :) ) : 
```
func MockExecer(logger *zap.SugaredLogger, kubeContext string) *helmexec.Execer {
	execer := helmexec.New(logger, kubeContext, &mockRunner{})
	return execer
}
```
but I am agree that it does not need to be exposed to other package if we can do another way in tests.

Reply:
Would it work if you made the return type `helmexec.Interface`, like `func MockExecer(logger *zap.SugaredLogger, kubeContext string) helmexec.Interface`?

Reply:
you are right, it's fixed

Reply:
Perfect! Thanks again for your support ☺️